examples/Dockerfile.scratch-functions-framework

```diff
@@ -0,0 +1,19 @@
+# Use this as the Dockerfile for your Dart Functions Framework apps. Examples:
+# https://github.com/GoogleCloudPlatform/functions-framework-dart/tree/main/examples
+FROM dart
+
+WORKDIR /app
+COPY pubspec.yaml /app/pubspec.yaml
```

Comment:
We probably should copy in both `pubspec.yaml` and `pubspec.lock`.

Reply:
Tracking in https://github.com/dart-lang/dart-docker/issues/4

Reply:
Addressed comment [here](https://github.com/dart-lang/dart-docker/issues/4#issuecomment-826788841).